src/docfx/publish/PublishItem.cs

```diff
@@ -20,6 +20,8 @@ internal class PublishItem
 
         public string MonikerGroup { get; set; }
 
+        public string ConfigMonikerRange { get; set; }
```

Comment:
```suggestion
        public string MonikerRange { get; set; }
```

Reply:
`ConfigMonikerRange` should be better and no confuse since there are several source of monikerRange: config, metadata and yaml header.